internal/kustomize/dependencies.go

```diff
@@ -0,0 +1,97 @@
+package kustomize
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// kustomization is the content of a kustomization.yaml file.
+type kustomization struct {
```

Comment:
is this copied from the kustomize repo? would it ever make sense to import their code directly (i haven't looked into how well their code is organized...)

Reply:
[It is](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/deploy/kustomize.go), but it doesn't look very importable unless I'm missing something.